src/atoms/Layout/index.js

```diff
@@ -0,0 +1,81 @@
+import React, { PropTypes } from 'react';
+import classnames from 'classnames';
+
+import styles from './layout.module.scss';
+
+function Layout ( props ) {
+  const {
+    children,
+    childCols,
+    breakpoint,
+    horizontalAlign,
+    verticalAlign
+  } = props;
+
+  let colsIdx = -1;
+
+  const kids = React.Children.map( children, ( child, idx ) => {
+    const length = !children.length ? 1 : children.length;
+    const childColsLength = childCols.length;
+
+    colsIdx = childColsLength == 1 || ( (idx + 1) > childColsLength && (idx + 1) % childColsLength === 1 ) ? -1 : colsIdx;
+
+    colsIdx++;
+
+    return (
+      <div
+        className={ classnames(
+          styles[`column-${ childCols[colsIdx] }`]
+        )}
+      >
+        { child }
+      </div>
+    )
+  });
+
+  const hAlign = 'h-' + horizontalAlign;
+  const vAlign = 'v-' + verticalAlign;
+
+  return (
+    <div>
+      <div
+        className={
+          classnames( styles.layout, styles[breakpoint], styles[hAlign], styles[vAlign] )
+        }
+      >
+        { kids }
+      </div>
+    </div>
+  )
+};
+
+Layout.propTypes = {
+  /**
+   * determines pixel breakpoint for columns and rows
+   */
+  breakpoint: PropTypes.string,
```

Comment:
So.. since you can't tie "ratios" to breakpoints, then it's really just deciding when the rows switch up to full-width columns. It seems like this is something that can be handled by the parent / consumer - perhaps we should leave this out of the interface?